launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
As commented on below, the instructions can be greatly simplified once we have a published package, which seems like the way to go.